_inc/lib/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This is the base class for every Core API endpoint Jetpack uses.
+ *
+ */
+abstract class Jetpack_Core_API_XMLRPC_Consumer_Endpoint {
+
+	/**
+	 * An instance of the Jetpack XMLRPC client to make WordPress.com requests
+	 *
+	 * @private
+	 * @var Jetpack_IXR_Client
+	 */
+	private $xmlrpc;
```

Comment:
Just a thought, but I'm curious about the naming here of `$xmlrpc` with respect to the comment about `Jetpack_IXR_Client` - what does that mean?

I didn't know what IXR meant, so I searched the codebase. To my surprise, I couldn't find it _anywhere_. I see it used a bunch, but never does it appear to be defined. I searched Google, and found what I think it probably means, as in, there's an XMLRPC library called IXR.

Maybe this is too broad of a thing to bring up here, but it would be nice to know what's going on. My original pause came from wondering why the naming didn't match the comment - is it XMLRPC or is it this IXR thing?